src/gluonts/model/forecast.py

```diff
@@ -24,67 +24,27 @@
 from gluonts.core.component import validated
 
 
-class LinearInterpolation:
-    """
-    Linear interpolation based on datapoints (x_coord, y_coord)
+def _linear_interpolation(
+    xs: np.ndarray, ys: np.ndarray, x: float
+) -> np.ndarray:
+    assert sorted(xs) == xs
```

Comment:
Can we complement the asserts with a text message?

Reply:
We could, but I don't see much value in it. The traceback should already contain enough information. Also, this function is only used internally.

Reply:
Sounds good.